configs/sst_kernel_rats-perf.yaml

```diff
@@ -20,6 +20,8 @@ data:
       - make
       - numactl-devel
       - python3-devel
+    limit_arches:
```

Comment:
That needs one more level of indentation, so it's under `rt-tests`. (Would do that for you, but can't edit your branch)